From 147d4eab0d696d8fb3b8bb2534a5a5e387631fa3 Mon Sep 17 00:00:00 2001
From: kbuild test robot <fengguang.wu@intel.com>
Date: Thu, 11 May 2017 09:48:31 +0800
Subject: [PATCH 1564/2094] drm/amdgpu/display: fix semicolon.cocci warnings

drivers/gpu/drm/amd/amdgpu/../display/dc/dce/dce_stream_encoder.c:411:23-24: Unneeded semicolon
drivers/gpu/drm/amd/amdgpu/../display/dc/dce/dce_stream_encoder.c:420:39-40: Unneeded semicolon

 Remove unneeded semicolon.

Generated by: scripts/coccinelle/misc/semicolon.cocci

CC: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
index a065e4a..daab81b 100644
--- a/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
+++ b/drivers/gpu/drm/amd/display/dc/dce/dce_stream_encoder.c
@@ -408,7 +408,7 @@ static void dce110_stream_encoder_dp_set_stream_attribute(
 	if (REG(DP_MSA_TIMING_PARAM1)) {
 		switch (output_color_space) {
 		case COLOR_SPACE_SRGB:
-			misc0 = misc0 | 0x0;;
+			misc0 = misc0 | 0x0;
 			misc1 = misc1 & ~0x80; /* bit7 = 0*/
 			break;
 		case COLOR_SPACE_SRGB_LIMITED:
@@ -417,7 +417,7 @@ static void dce110_stream_encoder_dp_set_stream_attribute(
 			break;
 		case COLOR_SPACE_YCBCR601:
 			misc0 = misc0 | 0x8; /* bit3=1, bit4=0 */
-			misc1 = misc1 & ~0x80; /* bit7 = 0*/;
+			misc1 = misc1 & ~0x80; /* bit7 = 0*/
 			if (crtc_timing->pixel_encoding == PIXEL_ENCODING_YCBCR422)
 				misc0 = misc0 | 0x2; /* bit2=0, bit1=1 */
 			else if (crtc_timing->pixel_encoding == PIXEL_ENCODING_YCBCR444)
-- 
2.7.4

